doxygen-awesome.css

```diff
@@ -1414,8 +1417,8 @@ div.memdoc, div.memproto, h2.memtitle {
 }
 
 div.memdoc {
```

Comment:
reason why i added `2px` to padding was, if i left it as it is, and then change the background color slightly lighter/darker to that of the main page :

![image](https://user-images.githubusercontent.com/90788311/206268308-61d02026-a57b-4b67-9e5f-76f7ec953680.png)